packages/@tinacms/toolkit/src/components/media/media-manager.tsx

```diff
@@ -498,8 +500,7 @@ const SyncModal = ({ close, syncFunc, folder, branch }) => {
         <ModalHeader close={close}>Sync Media</ModalHeader>
         <ModalBody padded={true}>
           <p>
-            {`This will copy all media from the \`${folder}\` folder on branch \`${branch}\` in your git repository to Tina Cloud. Are
-            you sure you would like to perform this action?`}
+            {`This will copy media assets from the \`${folder}\` folder on branch \`${branch}\` in your git repository to Tina Cloud's asset service for better performance.`}
```

Comment:
```suggestion
            {`This will copy media assets from the \`${folder}\` folder on branch \`${branch}\` in your git repository to Tina Cloud's asset service.`}
```
Nitpick, but "for better performance", makes it seem like it's an optional thing

Reply:
Updated the message